src/hotspot/share/gc/shenandoah/shenandoahNumberSeq.cpp

```diff
@@ -121,40 +121,68 @@ double HdrSeq::percentile(double level) const {
   return maximum();
 }
 
-// Merge this HdrSeq into hdr2: clear optional and on-by-default
-// Note: this method isn't intrinsically MT-safe; callers must take care
-// of any mutual exclusion as necessary.
-void HdrSeq::merge(HdrSeq& hdr2, bool clear_this) {
+void HdrSeq::add(HdrSeq& other) {
```

Comment:
Can we make `other` be a `const` reference? just to make it clear who is getting added to who.

Reply:
Yes, added!